vnext/Microsoft.ReactNative/QuirkSettings.idl

```diff
@@ -24,6 +24,9 @@ namespace Microsoft.ReactNative {
 
     // Runtime setting allowing Networking (HTTP, WebSocket) connections to skip certificate validation.
     static void SetAcceptSelfSigned(ReactInstanceSettings settings, Boolean value);
+
+    // Enable ability to use new Fabric UIManager within this instance.
+    static void SetEnableFabric(ReactInstanceSettings settings, Boolean value);
```

Comment:
this should be in an experimental interface until this is shippable

Reply:
Whats the way to mark an interface as experimental?

Reply:
you put the [experimental] attribute on the type or member function
see https://docs.microsoft.com/en-us/uwp/api/windows.foundation.metadata.experimentalattribute